flow/declarations/CardanoCrypto.js

```diff
@@ -2,114 +2,114 @@ import PasswordProtect from "../../js-cardano-wasm/js/PasswordProtect";
 
 declare module 'rust-cardano-crypto' {
   declare module.exports: {
-    loadRustModule(): Promise<void>,
-    Blake2b: {
+    loadRustModule(): Promise < void>,
```

Comment:
I’m not sure what happened here (auto formatter fail). I will restore this tomorrow.